src/test/java/core/basesyntax/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.OperationType;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private static OperationHandler handler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handler = new AddOperationHandler();
+        Storage.getFruits().put(new Fruit("apple"), 10);
+    }
+
+    @After
+    public void after() {
+        Storage.getFruits().clear();
+    }
+
+    @Test
+    public void testSupply_Ok() {
+        Transaction transaction = new Transaction(OperationType.SUPPLY, "apple", 50);
+        int expected = 50;
+        int actual = new AddOperationHandler().apply(transaction);
```

Comment:
can you use your field?
same in other handlers

Reply:
edited